docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 services:
   web:
     build: .
-    command: bash -lc "(rm -rf tmp/pids/ || true) && yarn install && bundle install --path vendor/bundle && bin/rails db:migrate db:seed RAILS_ENV=development && bundle exec foreman start -f Procfile.development"
+    command: bash -lc "(rm -rf tmp/pids/ || true) && yarn install && bundle exec foreman start -f Procfile.development"
```

Comment:
why are we removing the migration from here?

Reply:
I believe there are cases where you might want to boot the app without always running migrations. They are easily executed with `docker-compose exec` so I removed them here.